tests/auto-jenkins/start_rt_auto_jenkins.sh

```diff
@@ -22,7 +22,7 @@ elif [[ $HOSTNAME == tfe* ]]; then
 elif [[ $HOSTNAME == gaea* ]]; then
   export PATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/envs/ufs-weather-model/bin:$PATH
   export PYTHONPATH=/lustre/f2/pdata/esrl/gsd/contrib/miniconda3/4.8.3/lib/python3.8/site-packages
-elif [[ $HOSTNAME == *chadmin* ]] || [[ $HOSTNAME == *derecho* ]]; then
+elif [[ $HOSTNAME == *derecho* ]]; then
```

Comment:
Please remove the first *. should be `derecho*`

Reply:
done